native/src/main/java/io/ballerina/persist/nodegenerator/BalSyntaxTreeGenerator.java

```diff
@@ -564,31 +567,29 @@ private static Class createClientStreamClass(Entity entity, String className) {
                 AbstractNodeFactory.createLiteralValueToken(SyntaxKind.STRING_LITERAL, " ",
                         NodeFactory.createEmptyMinutiaeList(), NodeFactory.createEmptyMinutiaeList())), true);
 
-        clientStream.addMember(TypeDescriptor.getObjectFieldNodeWithoutExpression(BalFileConstants.KEYWORD_PRIVATE,
-                new String[]{},
-                TypeDescriptor.getStreamTypeDescriptorNode(
-                        SyntaxTreeConstants.SYNTAX_TREE_VAR_ANYDATA,
-                        TypeDescriptor.getOptionalTypeDescriptorNode(BalFileConstants.EMPTY_STRING,
-                                BalFileConstants.ERROR)),
-                BalFileConstants.ANYDATA_STREAM), true);
+        clientStream.addMember(NodeParser.parseStatement(ANYDATA_STREAM_STATEMENT), false);
+        clientStream.addMember(NodeParser.parseStatement(NULLABLE_ERROR_STATEMENT), false);
 
         Function initStream = new Function(BalFileConstants.INIT);
         initStream.addQualifiers(new String[]{BalFileConstants.KEYWORD_PUBLIC, BalFileConstants.KEYWORD_ISOLATED});
         initStream.addStatement(NodeParser.parseStatement(BalFileConstants.INIT_STREAM_STATEMENT));
-        initStream.addRequiredParameter(
-                TypeDescriptor.getStreamTypeDescriptorNode(
-                        SyntaxTreeConstants.SYNTAX_TREE_VAR_ANYDATA,
-                        TypeDescriptor.getOptionalTypeDescriptorNode(BalFileConstants.EMPTY_STRING,
-                                BalFileConstants.ERROR)),
-                BalFileConstants.ANYDATA_STREAM);
+        initStream.addStatement(NodeParser.parseStatement(SELF_ERR));
+        initStream.addRequiredParameter(NodeParser.parseTypeDescriptor("stream<anydata, error?>?"), ANYDATA_KETWORD);
+        initStream.addRequiredParameterWithDefault(NodeParser.parseTypeDescriptor("error?"), "err",
```

Comment:
Same.

Reply:
updated in [a7041fd](https://github.com/ballerina-platform/persist-tools/pull/31/commits/a7041fdbccf9572781cd185694e8cb15352d4e0b)